MarkBind/markbind
# Update contributors table and remove unnecessary margin top

**What is the purpose of this pull request?**




**Overview of changes:**
After the upstream change has been merged (https://github.com/all-contributors/cli/pull/307), the all-contributors tooling now generates HTML table with `<tbody>`, hence fixing the previous hydration issue in the about page. Thus,

- remove margin top as the page no longer needs extra space to deal with hydration issue
- regenerate the tables to apply the change
- add "review" to myself (as part of the testing to trigger the cli)

**Anything you'd like to highlight/discuss:**
The bot is also updated, so usage on GitHub UI will work correctly as well.

**Testing instructions:**
Check the about page and the README to see the content rendered correctly.
https://deploy-preview-2031--markbind-master.netlify.app/about.html
https://github.com/tlylt/markbind/blob/fix-contrib/README.md

Additionally, after the PR has been merged, can try to invoke the bot to see if the change is correct.



**Proposed commit message: (wrap lines at 72 characters)**
Update contributors table and remove unnecessary margin-top



---

**Checklist:** :ballot_box_with_check:
